### DISCUSSION TITLE
Are XCode Command Line Tools required for just development, or runtime/clients?

### DISCUSSION THREAD

**Discussion by adamjerickson** [nut-tree/nut.js#474] [Q&A] [answered]
Are xcode command line tools required for NutJS to run in general, for the machines doing the testing, or just when creating/coding?

**Comment by s1hofmann** [comment, marked as answer]
Hi @adamjerickson 👋 

they're not required to use nut.js.
If you are not necessarily recompiling some of the native addons, you shouldn't need them.
